tests/conftest.py

```diff
@@ -65,6 +66,18 @@ def __truediv__(self, other):
         return urljoin(self.base, other)
 
 
+@pytest.fixture
+def default_browser_options():
```

Comment:
Does it make sense to create a browser fixture with these options to handle `await browser.close()` inside such fixture?